pages/content-layout/without-header.page.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import AppLayout from '~components/app-layout';
+import ContentLayout from '~components/content-layout';
+import Container from '~components/container';
+import Toggle from '~components/toggle';
+import Flashbar from '~components/flashbar';
+import Header from '~components/header';
+import { Breadcrumbs } from '../app-layout/utils/content-blocks';
+import ScreenshotArea from '../utils/screenshot-area';
+import appLayoutLabels from '../app-layout/utils/labels';
+
+export default function () {
+  const [hasBreadcrumbs, setHasBreadcrumbs] = useState(false);
+  const [hasNotifications, setHasNotifications] = useState(false);
+  const [stickyNotifications, setStickyNotifications] = useState(true);
+  const [disableOverlap, setDisableOverlap] = useState(false);
+  return (
+    <ScreenshotArea gutters={false}>
+      <AppLayout
+        contentType="form"
+        ariaLabels={appLayoutLabels}
+        stickyNotifications={stickyNotifications}
+        notifications={
+          hasNotifications && (
+            <Flashbar
+              items={[
+                {
+                  type: 'success',
+                  header: 'Success message',
+                  statusIconAriaLabel: 'success',
+                },
+              ]}
+            />
+          )
+        }
+        breadcrumbs={hasBreadcrumbs && <Breadcrumbs />}
+        content={
+          <ContentLayout disableOverlap={disableOverlap}>
```

Comment:
What kind of use-cases do we have for Content layout without header? Can we make this demo page align more with that so it's less abstract?

Reply:
The use cases we have are pages without header and with a container